src/iperf_api.c

```diff
@@ -428,10 +428,8 @@ iperf_set_test_num_streams(struct iperf_test *ipt, int num_streams)
 static void
 check_sender_has_retransmits(struct iperf_test *ipt)
 {
-    if (ipt->sender && ipt->protocol->id == Ptcp && has_tcpinfo_retransmits())
-	ipt->sender_has_retransmits = 1;
-    else
-	ipt->sender_has_retransmits = 0;
+    if (ipt->sender && ipt->protocol->id == Ptcp)
```

Comment:
This is the right place.  I'm wondering if we should initialize `ipt->sender_has_retransmit` to zero just above this line?  Otherwise it's possible to run through the function without setting it...the original version of the code always set this value one way or another.

Reply:
Oh, you are looking at outdated code (I reset it cause saw undefined behaviour).